pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc-search-filters.adoc

```diff
@@ -0,0 +1,14 @@
+[id='search-filters_{context}']
+
+The Search tab allows you to apply a number of filters to your search that are available as drop down menus.
+ 
+.Procedure
+
+Use any combination of the following filters:
+
+* Select a Product name from the `Select a Product` menu.
+* Select a version number from the `Select a Version` menu.
+* Select a module type from the third menu (`All`, `Concept`, `Procedure`, or `Reference`).
+* Select a property from the fourth menu (`Uploaded date`, `Title`, `Updated date`, or `Module type`)
```

Comment:
Probably worth noting that this is for sorting purpose only (it's not a filter like the other 3) and only applies once the search is invoked.